benchmarks/perception/a5_resize/benchmark.yaml

```diff
@@ -0,0 +1,37 @@
+id: a5
+name: a5_resize
+description: |
+  A simple perception resize ROS robotics operation. Used to demonstrate a simple perception component [image_pipeline](https://github.com/ros-perception/image_pipeline) package.
+short: Perception resize ROS Component.
+graph: ../../../imgs/a5_resize.svg
```

Comment:
This file doesn't exist in this PR. I can see `a2_rectify.svg`. Can you produce the graph and modify it accordingly?